node-services/cards-reminder/src/domain/server-side/remindDatabaseService.ts

```diff
@@ -31,6 +31,10 @@ export default class RemindDatabaseService {
         return this.mongoDB.collection(this.remindersCollection).find({hasBeenRemind: false, timeForReminding: { $lte: new Date().valueOf() }}).toArray();
     }
 
+    public async getAllCardsToRemind(): Promise<any[]> {
+        return this.mongoDB.collection('cards').find({secondsBeforeTimeSpanForReminder: { $exists: true}}).toArray();
```

Comment:
To avoid uncessary processing , filter as well by endDate (endDate null or > current time)